input/.clang-format

```diff
@@ -0,0 +1,151 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Mozilla
```

Comment:
What's this file? (erroneous addition, or does it make sense to have it in the repo?)

Reply:
It's the clang-format config being used for the code formatting, so, yes ;).

Reply:
Code formatting by what ?

Reply:
Well, Clang's `clang-format` ;).